source/includes/_opportunities.md

```diff
@@ -0,0 +1,278 @@
+# Opportunities API
+
+Opportunities are records that you can use to track possible deals with your prospective customers.
+Opportunities can be associated with both companies and people in Apollo.
+
+## Create Opportunity
+
+> Sample request:
+
+```shell
+curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -d '{
+    "api_key": "YOUR API KEY HERE",
+    "owner_id":"5c10XXXXXXXXXXXXXXXXXXXX",
+    "name":"Opportunity Name",
+    "amount":"200",
+    "opportunity_stage_id":"5c14XXXXXXXXXXXXXXXXXXXX",
+    "closed_date":"2020-12-18",
+    "account_id":"5f06XXXXXXXXXXXXXXXXXXXX"
+}' "https://app.apollo.io/api/v1/opportunities"
+```
+
+> Sample response:
+
+```json
+{
+  "opportunity": {
+    "id": "5fdcXXXXXXXXXXXXXXXXXXXX",
+    "team_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+    "owner_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+    "salesforce_owner_id": null,
+    "amount": 200.0,
+    "closed_date": "2020-12-18T00:00:00.000+00:00",
+    "account_id": "5f06XXXXXXXXXXXXXXXXXXXX",
+    "description": null,
+    "is_closed": null,
+    "is_won": null,
+    "name": "Opportunity Name",
+    "stage_name": null,
+    "opportunity_stage_id": "5c14XXXXXXXXXXXXXXXXXXXX",
+    "salesforce_id": null,
+    "created_at": "2020-12-18T10:41:54.591Z",
+    "existence_level": "none",
+    "opportunity_contact_roles": []
+  }
+}
+```
+
+`POST https://api.apollo.io/v1/opportunities`
+
+| Parameter            | Description                                                                                                              |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| owner_id             | Owner ID . You can GET a list of possible users and their associated information from [Misc/Users](#get-a-list-of-users) |
+| name                 | Opportunity Name                                                                                                         |
+| amount               | The Amount of money involved in the Opportunity/ Deal                                                                    |
+| opportunity_stage_id | The ID of the current Stage (e.g. Prospecting, Closed Won, Closed Lost) of this Opportunity                              |
```

Comment:
Shouldn't we mention how `opportunity_stage_id` can be found(Similar to `owner_id`)?

Reply:
that's true but the opportunity_stages has no endpoint for viewing (only create and update endpoints are allowed)
and opportunity stages can also be gotten from the team object 

<img width="1769" alt="Screenshot 2020-12-23 at 19 29 40" src="https://user-images.githubusercontent.com/13007654/103027017-668d3900-4555-11eb-9a8a-bd8ad6d22b84.png">
<img width="843" alt="Screenshot 2020-12-23 at 19 29 23" src="https://user-images.githubusercontent.com/13007654/103027030-6ab95680-4555-11eb-9281-baccfd9cbaf6.png">

So possible solutions
1. an endpoint to get my team opportunities

Reply:
Try to mention that `opportunity_stage_id` is available at team object.

Reply:
something like this right?
<img width="709" alt="Screenshot 2020-12-23 at 19 51 42" src="https://user-images.githubusercontent.com/13007654/103028563-4f9c1600-4558-11eb-924f-30e56f6e6474.png">

Reply:
Should be good.